### PR TITLE
Website: add post about first meetup

### PR DESCRIPTION
**Pull request by wsdjeg**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I have updated the [Following-HADE](https://github.com/SpaceVim/SpaceVim/blob/master/wiki/en/Following-HEAD.md) page for this PR.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

This is our first meetup.

### CODE REVIEW

**Comment by icearith** [comment]
I feel like I should send you some photos of my SpaceVim and my laptop. It's such a shame that I didn't take a picture together with you guys.

**Comment by icearith** [comment]
I feel like I should send you some photos of my SpaceVim and my laptop. It's such a shame that I didn't take a picture together with you guys.

File: docs/_posts/2018-06-24-SpaceVim-meetup-hangzhou.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+1. where and how do you know about SpaceVim?
+
+2. which language do you use SpaceVim to development?
+
+3. what is issue do you encounter when using SpaceVim for such language?
+
+4. what is new features do you need in next verison of SpaceVim?
+
+5. the roadmap of next releas.
```
Comment (icearith) [inline docs/_posts/2018-06-24-SpaceVim-meetup-hangzhou.md]: Typo: releas

**Comment by wsdjeg** [comment]
@icearith yeah, that will be better, if you can send me some photos. I am too busy these days,  I will try to finish this post in the weekend.

**Pull request merged**